yarn.lock

```diff
@@ -2793,13 +2793,6 @@ bl@~1.1.2:
   dependencies:
     readable-stream "~2.0.5"
 
-block-stream@*:
```

Comment:
Is this file accidentally included? Or included on purpose?

Reply:
AIUI, yarn.lock is updated occasionally based on upstream package dependencies.
I can remove the yarn.lock changes in this PR, if desired, but we should definitely update yarn.lock regularly -- and if we don't merge the changes here, then every time people do a `yarn install` the same changes will enter through a different PR.